src/components/Sidebar/index.jsx

```diff
@@ -0,0 +1,156 @@
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import './Sidebar.css';
+
+import {
+  TiSocialGooglePlus,
+  TiSocialFacebook,
+  TiSocialTwitter,
+  TiSocialVimeo,
+  TiSocialPinterest,
+} from 'react-icons/ti';
+
+import { RxCross1 } from 'react-icons/rx';
+import { Button } from 'react-bootstrap';
+import { useSelector } from 'react-redux';
+import { getUser } from '../../redux/user/userSlice';
+
+const Sidebar = ({ isActive, handleHamburgerBtnClick }) => {
+  const user = useSelector(getUser);
+  console.log(user);
```

Comment:
- [ ] Kindly, remove this console.log debugging statement in order to improve the repo's cleanliness. 👍